### DISCUSSION TITLE
2008/03/06/%E5%85%AC%E5%8F%B8%E5%8D%81%E5%91%A8%E5%B9%B4%E5%BA%86%E5%85%B8

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#211] [Announcements]
# 2008/03/06/%E5%85%AC%E5%8F%B8%E5%8D%81%E5%91%A8%E5%B9%B4%E5%BA%86%E5%85%B8

今年是NI上海分公司成立的第十个年头。昨天，公司在举办了一次隆重的庆典。庆典分两部分：

https://blog.qizhen.xyz/2008/03/06/%E5%85%AC%E5%8F%B8%E5%8D%81%E5%91%A8%E5%B9%B4%E5%BA%86%E5%85%B8

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Bonnie
2008年11月6日 22:33 编辑
最近也在做十周年庆典活动的节目。无意中搜索到这里。
想知道你们的产品show效果好吗

回复
Qizhen
2008年03月21日 08:40 编辑
你说的这个方法我还真不知道，如果你打听出它确实可行，麻烦也告诉我一声:)我看到在DLL调用上大家遇到的问题还是很多的，我打算下面就详细说明介绍一下。

回复
浣熊爱上猫
2008年03月20日 09:40 编辑
大师大师，有人跟我说可以调用ReadProcessMemory,您觉得可行吗？

回复
Qizhen
2008年03月18日 18:10 编辑
如果返回的指针是函数的参数，只要在 Call Library Function Node 中把这个参数的类型设为数组就行了。如果是函数返回值，就比较麻烦了。LabVIEW 没办法直接处理这样的指针。你只能自己用C语言再写一个函数，把dll中的函数包装一下，把你需要的指针用函数参数的形式返回给 LabVIEW。

回复
浣熊爱上猫
2008年03月18日 18:02 编辑
大师您好！我现在在用labview调用ccd相机的dll.现在遇到的问题是：调用动态库，返回一个指针，指向我所需要的数据的首地址，我怎样才能得到这个数据的一维数组呢？
希望您能帮我解答一下，不胜感激！！

回复
ruirui
2008年03月10日 09:46 编辑
照片~~照片~~我们要照片~~

回复
茂
2008年03月8日 09:59 编辑
要是早两个月毕业的话我就能赶上庆典了,期待良师上传照片:)

回复
磚頭MAN
2008年03月7日 18:20 编辑
NI? National Instruments?

回复
Fu
2008年03月7日 16:26 编辑
You are NI man…

回复
秋
2008年03月7日 08:34 编辑
速传片片~~~

回复
